Index: __init__.py
===================================================================
--- __init__.py	(revision 904)
+++ __init__.py	(working copy)
@@ -27,11 +27,7 @@
     :license: BSD, see LICENSE for more details.
 """
 __docformat__ = 'restructuredtext en'
-try:
-    __version__ = __import__('pkg_resources') \
-        .get_distribution('Jinja2').version
-except:
-    __version__ = 'unknown'
+__version__ = '2.4.1'
 
 # high level interface
 from jinja2.environment import Environment, Template
